### ISSUE TITLE
[Design Enhancement]

### ISSUE THREAD

**Issue by prikiser** [CDCgov/ReportVision#394]
**Standardization of table**

- [ ] Table columns are name, date created, and facility no zebra stripes
- [ ] If possible, move run data CTA to the top and out of the table

[Figma design link
](https://www.figma.com/design/1FW8ZSDV90q2wpRPOADE9r/ReportVision?node-id=1637-12246&t=PbjaFODxjtEF7FtR-1)

**Comment by prikiser** [comment]
@bora-skylight I added this as an enhancement. Please feel free to change it into whatever you think is appropriate. It may be two stories as this change would require us adding [Facility](https://www.figma.com/design/1FW8ZSDV90q2wpRPOADE9r/ReportVision?node-id=1327-18963&t=sVhF2oc7ZEPriY03-1) as a data point for when the user saves the template (I'm not sure if it needs to be its own story).

**Comment by bora-skylight** [comment]
@prikiser can you please update this ticket with the latest version of the table, which also includes the Condition column

**Comment by bora-skylight** [comment]
Work on this ticket is blocked on having access to the Templates API and it being connected to the newly created database.

**Comment by bora-skylight** [comment]
Dependencies: #439 and #266 being completed
